documentation/12-factor-app-devon4j.asciidoc

```diff
@@ -0,0 +1,96 @@
+:toc: macro
+toc::[]
+:idprefix:
+:idseparator: -
+
+= 12-factor-app with devon4j
```

Comment:
The images are nowhere referenced.
Further `12-factor-app-config.JPG` only contains source-code that can be included directly as text into asciidoc.